src/test/interface/arguments/argument_configuration_test.cpp

```diff
@@ -106,6 +106,10 @@ TEST_F(StanGmArgumentsConfiguration, TestMethod) {
       
       if (argument.length() == 0) 
         continue;
+
+      // kludge - generate_quantities method doesn't fit test paradigm
+      size_t pos = 0; 
```

Comment:
Replace 111 and 112 with

```
if (argument.find("generate") != std::string::npos) continue;
```

because `pos` isn't used anywhere later.

Reply:
done